cg/meta/workflow/nf_analysis.py

```diff
@@ -472,3 +474,20 @@ def report_deliver(self, case_id: str) -> None:
         LOG.info(
             f"Writing deliverables file in {self.get_deliverables_file_path(case_id=case_id).as_posix()}"
         )
+
+    def store_analysis_housekeeper(self, case_id: str, dry_run: bool = False) -> None:
+        """Store a finished nextflow analysis in Housekeeper and StatusDB"""
+
+        try:
+            self.status_db.verify_case_exists(case_internal_id=case_id)
+            self.trailblazer_api.is_latest_analysis_completed(case_id=case_id)
+            self.verify_deliverables_file_exists(case_id=case_id)
+            self.upload_bundle_housekeeper(case_id=case_id, dry_run=dry_run)
+            self.upload_bundle_statusdb(case_id=case_id, dry_run=dry_run)
+            self.set_statusdb_action(case_id=case_id, action=None, dry_run=dry_run)
+        except ValidationError as error:
+            LOG.warning("Deliverables file is malformed")
+            raise error
```

Comment:
```suggestion
            raise StoreHouseKeeperError(""Deliverables file is malformed")
```

Reply:
You'll need to create the "StoreHousekeeperError" ,  name is subjective. There is perhaps a better alternative.